pinot-spi/src/main/java/org/apache/pinot/spi/utils/builder/ControllerRequestURLBuilder.java

```diff
@@ -626,6 +626,11 @@ public String forPauseStatus(String tableName) {
     return StringUtil.join("/", _baseUrl, "tables", tableName, "pauseStatus");
   }
 
+  public String forValidDocIdsMetadata(String tableName, String validDocIdsType) {
```

Comment:
nit: maybe allow specifying `null` for `validDocIdsType` to use the default value?

Reply:
pinot-spi doesn't include pinot-common so can't add ValidDocIdsType.SNAPSHOT here. and didn't want to use a String version of it as that makes it harder to find when required to change defaults.